Animations/Animation.js

```diff
@@ -37,20 +22,30 @@ var makeTransition = function (displayName, transitions) {
       // componentDidEnter FUNCTION: use to define actions to run after an animation has entry completed
       // componentDidLeave FUNCTION: use to define actions to run after an animation has leave completed
   return React.createClass({
-    componentWillEnter: function (callback) {
-      setAnimationDomNode('enter', callback);
+    // setAnimationDomNode FUNCTION: function to handle manipulating and setting a TransitionGroup in the DOM
+      //@param action STRING: the transition key you are looking to affect
+      //@param callback FUNCTION: callback attribute passed in from the containing function to act on function completion
+    setAnimationDomNode: function (action, callback) {
+      this.el = this.getDOMNode();
```

Comment:
Rather than storing `this.getDOMNode()` in `this.el`, store it in a variable not attached to this.  Two reasons: 1) that way we aren't unnecessarily mutating `this` 2) we don't need to keep doing a lookup of `el` on `this` and can instead just access a variable in memory.